pkg/kubelet/lifecycle/predicate_test.go

```diff
@@ -86,8 +86,8 @@ func TestRemoveMissingExtendedResources(t *testing.T) {
 		nodeInfo := schedulerframework.NewNodeInfo()
 		nodeInfo.SetNode(test.node)
 		pod := removeMissingExtendedResources(test.pod, nodeInfo)
-		if !reflect.DeepEqual(pod, test.expectedPod) {
-			t.Errorf("%s: Expected pod\n%v\ngot\n%v\n", test.desc, test.expectedPod, pod)
+		if diff := cmp.Diff(pod, test.expectedPod); diff != "" {
```

Comment:
this has the wrong order. It should be `(test.expectedPod, pod)` to match `(-want,+got)`

Reply:
done